cluster-classification/script.whizzml

```diff
@@ -0,0 +1,70 @@
+;; Given a cluster, creates a batch centroid and uses the output
+;; dataset where each instance is labeled by centroid to create an
+;; ensemble (random forest), and returns the importance of each field
+;; averaged over the ensemble.
+
+;; Given a cluster, returns a batchcentroid id created from that
+;; cluster whose output dataset is the original cluster dataset
+;; appended with the centroids the instances are part of.
+
+(define (label-by-cluster cl-id) 
+  (let (cl-ds (get-in (fetch cl-id) ["dataset"]) 
```

Comment:
we're just releasing a new version of whizzml where maps and lists can be used as procedures, so that the idiomatic way of writing the above will become:

```
((fetch cl-id) "dataset")
```

it's fine if you prefer to send a PR afterwards modifying that, but you can also wait a day and fix it in this PR.

If we go with the current version, the currently idiomatic way would be using `get`:

```
(get (fetch cl-id) "dataset")
```